docs/try-flink/python_table_api.md

```diff
@@ -23,161 +23,9 @@ specific language governing permissions and limitations
 under the License.
 -->
 
-This walkthrough will quickly get you started building a pure Python Flink project.
```

Comment:
If you're moving the content, please just delete this page and setup a redirect.

Reply:
Hi, the Python DataStream tutorial will also be added to this page(FLINK-18984). I was thinking about keeping this page in `Try Flink` and add both Python Table API tutorial link and Python DataStream API tutorial link in it. 

What do you think?

Reply:
that's fine then.